virtool/subtractions/data.py

```diff
@@ -149,16 +183,14 @@ async def create(
                 "ready": False,
                 "space": {"id": space_id},
                 "upload": data.upload_id,
-                "user": {"id": user_id},
+                "user": {"id": user_id}
             }
         )
 
-        subtraction = await self.get(document["_id"])
-
         await self.data.jobs.create(
             "create_subtraction",
             {
-                "subtraction_id": subtraction.id,
+                "subtraction_id": subtraction_id or default_subtraction_id,
```

Comment:
Should use the returned `document["_id]`. As written a job could be created for a subtraction that may or may not even exist or be related to the one that is being created.